<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KateSnippetRepository</class>
 <widget class="QWidget" name="KateSnippetRepository">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Snippet Repository:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="lstSnippetFiles"/>
   </item>
   <item>
    <widget class="KPushButton" name="btnNew">
     <property name="text">
      <string>New snippet file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KUrlRequester" name="urlSource"/>
     </item>
     <item>
      <widget class="KPushButton" name="btnCopy">
       <property name="text">
        <string>Copy to repository</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
